rbac/server/api/roles.py

```diff
@@ -85,14 +83,15 @@
     response = await roles_query.roles_search_duplicate(
         request.app.config.DB_CONN, role_title
     )
-    if request.json.get("metadata") is None or request.json.get("metadata") == {}:
-        set_metadata = {}
-    else:
-        set_metadata = request.json.get("metadata")
-    set_metadata["sync_direction"] = "OUTBOUND"
     if not response:
         txn_key, txn_user_id = await utils.get_transactor_key(request)
         role_id = str(uuid4())
+
+        if request.json.get("metadata") is None or request.json.get("metadata") == {}:
```

Comment:
I think the `or request.json.get("metadata") == {}` is redundant, is there a weird reason that we need it?

Reply:
Not that I can think of. Removing the additional condition now.